src/World3/pollution/plots.jl

```diff
@@ -26,6 +26,62 @@ function variables_31()
 end
 
 
+function fig_26()
+    @named pp = persistent_pollution()
+    @named pd = pollution_damage()
+
+    variables = [
+        (pp.ppgr,  0, 1e10, "ppgr"),
+        (pp.ppapr, 0, 1e9, "ppapr"),
+        (pp.ppasr, 0, 1e9, "ppasr"),
+        (pp.ppolx, 0, 24,   "ppolx"),
+        (pp.ahl,   0, 40,   "ahl"),
+        (pd.lmp,   0, 1,   "lmp"),
+        (pd.lfdr,  0, 0.5, "lfdr"),
+    ]
+
+    initialisations_6_26 = getinitialisations()
+    initialisations_6_26[:ppol] = 0
+
+    system = historicalrun(inits=initialisations_6_26)
+
+    new_equations = equations(system)
+    new_equations[31] = pp.ppgr ~ 1e10 * clip(1, 0, t, 1920) * clip(0, 1, t, 1922)
+
+    @named new_system = ODESystem(new_equations)
+    solution_6_26 = solve(new_system, (1900, 2100))
```

Comment:
since it's an internal process, you can just call `solution` instead of `solution_6_26`.